pkgs/tools/misc/mkbootimage/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libelf
+, pcre
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mkbootimage";
+  version = "2.3dev";
```

```suggestion
  version = "2.3-unstable-2022-05-26";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming